src/Soil-Core/Soil.class.st

```diff
@@ -140,3 +140,28 @@ Soil >> path [
 Soil >> path: aString [ 
 	path := aString asFileReference
 ]
+
+{ #category : #refactorings }
+Soil >> renameClassNamed: oldName to: newName [
+	| transaction behaviorDescription objectId |
+
+	"we load the behaviorDescription of the oldName, change the name and commit"
+
+	transaction := self newTransaction.
+
+	objectId := self behaviorRegistry
```

Comment:
Regarding version bump: The mechanic is so that the name always points to the newest version of a behavior description. We decided to keep the object id for a behavior description and introduce versions. So taking that at this point it is the newest and a simple call to #incrementVersion should be sufficient to have the version update

Reply:
done